src/assets/data/region/0.json

```diff
@@ -52,6 +52,14 @@
     "name": "M",
     "description": "Museums Fahrt"
   },
+  "44": {
```

Comment:
for which kind of tram are those two line numbers used?

Reply:
These two are replacement lines for lines 4 and 6, see https://www.dvb.de/de-de/meta/aktuelle-meldungen/bau_wehlener-srasse_24 .